app/views/play/level/tome/Problem.coffee

```diff
@@ -37,7 +37,12 @@ module.exports = class Problem
       @createdBy = 'web-dev-iframe'
       # TODO: Include runtime/transpile error types depending on something?
 
-    @message = @translate(@message)
+    camelToSnake = (str) ->
+      str.replace(/[A-Z]/g, (letter) -> "_#{letter.toLowerCase()}");
```

Comment:
We could use [the `_.string.underscored` method](https://gabceb.github.io/underscore.string.site/#underscored) here instead of writing our own? This confusingly adds a leading underscore when the error code (always?) starts with a capital letter, which isn't snake case, but we just happened to also need a leading underscore.

Reply:
yeah we always have error code starts with a capital letter and we need that leading underscore,  and that's why i write this 😂  we can use underscored. sure